### ISSUE TITLE
Geopoints not in historical centers

### ISSUE THREAD

**Issue by bmschmidt** [cestastanford/historical-us-city-populations#1]
I e-mailed Jason about this a few months ago, but just to note it formally (and my apologies if it comes off pedantic):

For most cities, the geocoded locations included seem to be some sort of centroid or bounding-box-middle for the city.  For historical mapping, this can be problematic because often the centroid was outside city limits (and very frequently, in a different **county**) for the time period mapped. For example, for New York City, the two available points are: `40.6642738,-73.9385004`, and `40.7820014953613,-73.8327026367`. One of these is in Queens, the other in Brooklyn; both were in a different county than New York City until the late 19th century.

It would be difficult to calculate a centroid for each year. A better single point is the [USGS](http://earthexplorer.usgs.gov/) location, which tends to be city hall or some other downtown location which in most cases has always been within city limits. (There may be exceptions to this, but probabl many fewer). [Geonames.org](http://www.geonames.org/) frequently has these--I may post a push with these edits at some point, if useful.

Some cities that I know to be affected for some portion of the 19th century (because they play poorly with census county data I've tried to use.):
1. New York
2. Boston
3. Denver
4. Philadelphia

And probably more.

**Comment by cartosattva** [comment]
Yes, this is an issue. The original census lat/longs were developed for modern regional mapping and they informed me that they are not reliable for large scale mapping, let along historical accuracy. I quickly added the Bing-derived coordinates but these are only sometimes more reliable.

@bmschmidt it would be great if you could do some work on the coordinates at some point!

In theory we could add time-dependent county names and coordinates but I'm reluctant to expand our scope unless someone is willing to take it on. The easier task is to add most reasonable present-day USGS or GNIS coordinate.
